web/yo/app/scripts/controllers/nav.js

```diff
@@ -23,12 +23,13 @@ angular.module('oncokbApp')
             filterTabs.push('genes');
             filterTabs.push('queues');
             filterTabs.push('therapies');
-            if ($rootScope.me.admin) {
+            filterTabs.push('tools');
+            if ($rootScope.me && $rootScope.me.admin) {
                 filterTabs = _.union(filterTabs, ['variant', 'tools', 'feedback']);
```

Comment:
Are we supposed to remove `tools` in here?

Reply:
Tools is only available for admins

Reply:
So we should remove line 26 right?

Reply:
Ah yes, let me do that.